src/vma/dev/ring_eth_cb.cpp

```diff
@@ -233,6 +224,20 @@ int ring_eth_cb::allocate_umr_mem()
 	}
 
 	pad_len = (m_stride_size * count) - net_len - m_hdr_len - m_payload_len;
```

Comment:
Consider to set padding address to data address (It should help with debugging/wrong expected packet size)